diff --git a/hlslang/MachineIndependent/Intermediate.cpp b/hlslang/MachineIndependent/Intermediate.cpp
index 02637ce..cfa8dd4 100644
--- a/hlslang/MachineIndependent/Intermediate.cpp
+++ b/hlslang/MachineIndependent/Intermediate.cpp
@@ -12,6 +12,7 @@
 #include "ParseHelper.h"
 #include <float.h>
 #include <limits.h>
+#include <algorithm>
 
 static TPrecision GetHigherPrecision (TPrecision left, TPrecision right) {
 	return left > right ? left : right;
